lifecycle/manager.go

```diff
@@ -137,9 +137,9 @@ func (lcs *LifecycleServicesManager) DistributeReleaseBundle(rbDetails lifecycle
 	return distributeBundleService.Distribute()
 }
 
-func (lcs *LifecycleServicesManager) RemoteDeleteReleaseBundle(rbDetails lifecycle.ReleaseBundleDetails, params lifecycle.ReleaseBundleRemoteDeleteParams) error {
+func (lcs *LifecycleServicesManager) RemoteDeleteReleaseBundle(rbDetails lifecycle.ReleaseBundleDetails, params lifecycle.ReleaseBundleRemoteDeleteParams, newReleaseBundleApiSupported bool) error {
```

Comment:
```suggestion
func (lcs *LifecycleServicesManager) RemoteDeleteReleaseBundle(rbDetails lifecycle.ReleaseBundleDetails, params lifecycle.ReleaseBundleRemoteDeleteParams, isNewReleaseBundleApiSupported bool) error {
```

Reply:
nit pick: you ca change all the references of newReleaseBundleApiSupported to make it more redable